dolphinscheduler-dao/src/main/resources/sql/upgrade/3.0.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -495,6 +495,104 @@ delimiter ;
 CALL alter_t_ds_task_instance_col_log_path;
 DROP PROCEDURE alter_t_ds_task_instance_col_log_path;
 
+-- Dealing with table name case issues
+drop PROCEDURE if EXISTS qrtz_table_name_lowercase_to_uppercase;
+delimiter d//
+CREATE PROCEDURE qrtz_table_name_lowercase_to_uppercase()
+BEGIN
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.STATISTICS
+        WHERE TABLE_NAME='qrtz_blob_triggers'
```

Comment:
When initializing the table structure, it will follow the configuration of the mysql library to create it, and there will be no differences in table naming between different versions.

The reason for this issue may be an exception caused by the user manually modifying the table naming method of the MySQL library after initializing the table structure, so I don't think this PR is necessary.